db/migrations/20210320210856_partial_table.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
Update the migration to not have partials in the template table.

Reply:
Are you suggesting migrate the `*/partial` and `*/image` rows into the partial table? I hadn't considered that, though that's definitely something we'll need to do.

Reply:
Yes, suggestion to migrate partials and images to the partial table.